.vitepress/theme/components/ShareFeedback.vue

```diff
@@ -0,0 +1,252 @@
+    v-model:show="openModal"
+    described-by="share-feedback-description"
+    @after-close="onAfterClose"
+  >
+    <div class="feedback-card shadow-lg flex flex-col">
+      <div class="feedback-card_header flex items-center">
+        <div
+          :id="api.labelledBy"
+          class="feedback-card_title flex-1"
+        >
+          Share feedback
+        </div>
+
+        <SButton
+          size="sm"
+          type="action"
+          @click="api.close()"
+        >
+          <template #icon>
+            <IconClose />
+          </template>
+        </SButton>
+      </div>
+
+      <template v-if="success">
+        <div class="p-4 flex items-center space-x-4">
+          <IconCheck class="text-3xl feedback-card_check" />
+          <div>Thank you for sharing your feedback!</div>
+        </div>
+
+        <div class="flex flex-row-reverse p-4">
+          <SButton @click="api.close()">
+            Close
+          </SButton>
+        </div>
+      </template>
+
+      <div
+        v-else
+        class="flex-1 overflow-y-scroll"
+      >
+        <div class="p-4 space-y-4">
+          <p
+            id="share-feedback-description"
+            class="text-sm"
+          >
+            Please take a moment to help us improve the Iroha 2 Documentation. We take your input very seriously.
+          </p>
+
+          <div>
+            <fieldset class="space-y-1">
+              <legend class="field-label">
+                What kind of feedback?*
+              </legend>
+
+              <div
+                v-for="value of KINDS"
+                :key="value"
+                class="flex space-x-2 items-center"
+              >
+                <input
+                  :id="`feedback-kind-${value}`"
+                  v-model="feedbackKind"
+                  class="max-w-min"
+                  :value="value"
+                  type="radio"
+                  name="feedback-kind"
+                >
+                <label
+                  :for="`feedback-kind-${value}`"
+                  class="flex-1 text-sm"
+                >{{ KINDS_LABELS[value] }}</label>
+              </div>
+            </fieldset>
+          </div>
+
+          <div>
+            <label
+              for="feedback-input-text"
+              class="field-label"
+            >Feedback*</label>
+
+            <textarea
+              id="feedback-input-text"
+              v-model="feedbackText"
+              placeholder="What can we do to improve the overall documentation browsing experience?"
+              rows="5"
+            />
+          </div>
+
+          <div>
+            <label
+              for="feedback-input-contact"
+              class="field-label"
+            >
+              <i>(optional)</i> Email address or any other way to contact you (Discord, Telegram)
+            </label>
+
+            <input
+              id="feedback-input-contact"
+              v-model="contact"
+              placeholder="Contact information"
```

Comment:
this might be better

```suggestion
              <i>(optional)</i> Contact information
            </label>

            <input
              id="feedback-input-contact"
              v-model="contact"
              placeholder="Email address, Discord, or Telegram"
```